packages/toolpad-app/src/utils/components.ts

```diff
@@ -0,0 +1,8 @@
+import type * as appDom from '../appDom';
```

Comment:
Kinda feels like this logic belongs under `./src/toolpadComponents` so that we can reserve `utils` for things that are toolpad agnostic